contrib/test_jenkins.sh

```diff
@@ -73,6 +73,7 @@ MAKEP="make -j${parallel_jobs}"
 export AUTOMAKE_JOBS=$parallel_jobs
 
 have_ptrace=$(capsh --print | grep 'Bounding' | grep ptrace || true)
+have_strace=$(strace -V || true)
```

Comment:
My only concern is differences between versions of strace.
For example in the MAN the option --version exists but on strace running on Jazz it's not exists.
I used `-V ` because it's mentioned in the MAN and also available on Jazz.

We can do a different check if you have different idea, for example
`whereis strace | grep \/ || true` (will not check if strace running but if the path exists it's probably avaialble.)

Reply:
Do you have any specific reason to believe these will be some setup where `-V` is unavailable?

Reply:
I didn't encounter with such setup or MAN that not include the -V option. This is why I checked it this way.
I just mentioned that for you to consider.

Reply:
IMO the concern regarding the potential absence of the `-V` flag is not so reasonable, so let's stick with it for now.